learn/app-development/deployment/build-with-docker.md

```diff
@@ -12,9 +12,9 @@ For creating a Docker container, you create a Docker image by building a Dockerf
 
 |Description|Version|
 |---|---|
-|Java |1.8|
```

Comment:
https://pr-649.d2ulx3keu6fdjg.amplifyapp.com/learn/app-development/deployment/building-with-maven
change here also..
Should we publish two docker files for 10x and 11.x ?